DependencyInjection/EzSystemsTweetFieldTypeExtension.php

```diff
@@ -18,8 +18,16 @@ class EzSystemsTweetFieldTypeExtension extends Extension implements PrependExten
 {
     public function prepend(ContainerBuilder $container)
     {
-        $config = Yaml::parse(file_get_contents(__DIR__.'/../Resources/config/ez_field_templates.yml'));
-        $container->prependExtensionConfig('ezpublish', $config);
+        $configDirectoryPath = __DIR__.'/../Resources/config';
```

Comment:
I don't think that it is regulated by PSR, but the Symfony does it the other way (http://symfony.com/doc/current/contributing/code/standards.html#structure, second point: "Add a single space around binary operators (==, &&, ...), **with the exception of the concatenation (.) operator**;"). What do you think?